hs_core/irods.py

```diff
@@ -140,7 +148,8 @@ def create_ticket(self, user, path=None, write=False, allowed_uses=1):
         formatted = timeout.strftime("%Y-%m-%d.%H:%M")
         istorage.session.run('iticket', None, 'mod', ticket,
                              'expires', formatted)
```

Comment:
@alvacouch The iticket mod command parameter appears to be ```expire``` rather than ```expires``` according to iRODS documentation https://docs.irods.org/4.1.3/icommands/tickets/. This is important to check to make sure, since otherwise, the ticket will not expire as specified, which would create security issues.